recipes-cfengine/cfengine-masterfiles_3.21.0.bb

```diff
@@ -25,16 +25,18 @@ SRC_URI = "https://cfengine-package-repos.s3.amazonaws.com/tarballs/${BP}.tar.gz
 #SRC_URI[md5sum] = "5df2f85c75efc351ffadebcc11046a98"
 SRC_URI[sha256sum] = "013ebe68599915cedb4bf753b471713d91901a991623358b9a967d9a779bcc16"
 
-inherit autotools
+inherit autotools-brokensep
 
 export EXPLICIT_VERSION="${PV}"
 
-EXTRA_OECONF = "--prefix=${datadir}/cfengine"
+do_configure() {
```

Comment:
This section just doesn't feel right. Have you looked at the environment that is being passed into this run? My guess is that a raw `./configure` is even more prone to break. Wouldn't passing `--prefix` and `--workdir` as in the other recipe work here too? If you want to avoid duplication, you can also put `/var/cfengine` into a variable.

Reply:
Right. I had the same thought. I will try this :+1: